components/suggestion/command_provider/app_command_parser/app_command_parser.test.ts

```diff
@@ -443,11 +443,6 @@ describe('AppCommandParser', () => {
     });
 
     describe('getSuggestions', () => {
-        test('just the app command', async () => {
-            const suggestions = await parser.getSuggestions('/jira');
-            expect(suggestions).toEqual([]);
-        });
-
```

Comment:
Why was this removed?

Reply:
Because this is not handle by `parser.getSuggestions` but by `parser.getBaseSuggestions` and with the latest changes the error this generated was being surfaced.

Reply:
I have just discovered this is not true. I will recover this test and fix the issue.

Reply:
After looking it further, I was right at the beginning. The getSuggestions should never be called before the base command is complete. Before that we usually call getsuggestionbase. The moment we add a space, it starts using the getsuggestions function.